### PR TITLE
Mkdocs

### PR DESCRIPTION
**Pull request by beobillly**
#195 
Hey this little pr is the answer to "Create the official website" with the markdown files in docs repository.
I use mkdocs (instead of sphinx) to build the website !
I test the deployment on readthedocs website, enjoy it : https://learn-ocaml.readthedocs.io/en/latest/
I'll ask you to test the site and the links on it to verifiy that there are no broken links !

Actually this pr is working with my learn-ocaml branch, I submit it as it to show you the working deployment, if you find it cool and your test on it are okay, some files must be changed before merging (just one line :+1: )

### CODE REVIEW

**Comment by yurug** [comment]
Please write better error messages.

**Comment by beobillly** [comment]
well, I think the links was not broken but it was a versionning problem from readthedocs which may the index page links not working correctly.
I'm sorry for the numberous commits and their messages.
Normally, it's resolved, but that kind of problem may occur again if we make some versionning modifications in readthedocs website

**Comment by beobillly** [comment]
the new temporary link is : https://learn-ocaml.readthedocs.io/en/mkdocs/

File: mkdocs.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+site_name: "learn-ocaml"
+
+repo_name: GitHub
+repo_url: https://github.com/beobillly/learn-ocaml
```
Comment (beobillly) [inline mkdocs.yml]:
```suggestion
repo_url: https://github.com/ocaml-sf/learn-ocaml
```

**Pull request merged**